app/helpers/admin_helper.rb

```diff
@@ -0,0 +1,2 @@
+module AdminHelper
+end
```

Comment:
if this helper are empty please remove it

Reply:
done